wallet-api/src/Wallet/Emulator/Types.hs

```diff
@@ -215,6 +215,11 @@ data EmulatorEvent =
 instance FromJSON EmulatorEvent
 instance ToJSON EmulatorEvent
 
+-- | Delete all 'EventHandler' values that are registered for an
+--   'EventTrigger'.
+deleteHandlers :: MonadState WalletState m => EventTrigger -> m ()
+deleteHandlers t = modify (over triggers (set (at t) Nothing))
```

Comment:
This makes me notice another deficit: we should be able to register multiple handlers for the same trigger! Which of course means they need to be cancellable separately, so they need some other kind of handle.

Reply:
We can register multiple handlers already (they will be `>>`'d). And the ones that don't reschedule themselves are cancelled.